src/navigations/HomeBottomTabs.js

```diff
@@ -106,24 +106,9 @@ function HomeBottomTabs({navigation}) {
     // Should the initial notification be popped automatically
     // default: true
     popInitialNotification: true,
-    requestPermissions: true
+    requestPermissions: false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `requestPermissions: true` to `requestPermissions: false` means that the app will no longer request notification permissions automatically at startup. This could potentially lead to issues if the user has not granted these permissions manually, as the app may not be able to send notifications. Please ensure that there is a manual prompt for the user to grant these permissions elsewhere in the code.

```diff
-    requestPermissions: true
+    requestPermissions: false
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
yes this is intentional